apps/heft/UPGRADING.md

```diff
@@ -2,41 +2,89 @@
 
 ### Heft 0.32.0
 
-This release of Heft removed the Jest plugin from the `@rushstack/heft` package
-and moved it to it's own package (`@rushstack/heft-jest-plugin`). To re-include
-Jest support in a project, include a dependency on `@rushstack/heft-jest-plugin`
-and add the following option to the project's `config/heft.json` file:
-
-```JSON
-{
-  "heftPlugins": [
-    {
-      "plugin": "@rushstack/heft-jest-plugin"
-    }
-  ]
-}
-```
+Breaking change for Jest:  This release of Heft enables rig support for Jest config files.
+It also reduces Heft's installation footprint by removing the Jest plugin from `@rushstack/heft`
+and moving it to its own package `@rushstack/heft-jest-plugin`.  As a result, Jest is now
+disabled by default.
+
+To reenable Jest support for your project, follow these steps:
```

Comment:
re-enable

Reply:
Naw... In fact I dug my heels in and removed `re-include` from the Webpack note 😆